guides/shipments/shipping.md

```diff
@@ -132,49 +197,48 @@ To achieve this setup you need the following configuration:
 
 With the above configuration, a customer shipping to the US would see the USPS Ground shipping option presented to them at checkout, while a customer shipping to the EU would see the FedEx option.  Shipping rate would be calculatd at checkout according to the calculator rules.
 
+### Advanced setup
 
-### Advanced Setup
-
-Consider you sell products to a single zone (US) and you ship from 2 locations (Stock Locations):
+Consider you sell products to a single zone (U.S.) and you ship from 2 locations (Stock Locations):
 
-* New York
-* Los Angeles
+- New York
+- Los Angeles
 
 and you work with 3 carriers (Shipping Methods):
 
-* FedEx
-* DHL
-* US postal service
+- FedEx
+- DHL
+- US postal service
 
 and your products can be classified into 3 Shipping Categories:
 
-* Light
-* Regular
-* Heavy
+- Light
+- Regular
+- Heavy
 
 and their pricing is as follow:
 
 FedEx charges:
 
-* $10 for all light items regardless of how many you have
-* $2 per regular item
-* $20 for the first heavy item and $15 for each additional one
+- $10 for all light items regardless of how many you have
+- $2 per regular item
+- $20 for the first heavy item and $15 for each additional one
 
 DHL charges:
 
-* $5 per item if it's light or regular
-* $50 per item if it's heavy
+- $5 per item if it's light or regular
+- $50 per item if it's heavy
 
 USPS charges:
 
-* $8 per item if it's light or regular
-* $20 per item if it's heavy
+- $8 per item if it's light or regular
+- $20 per item if it's heavy
 
 To achieve this setup you need the following configuration:
 
-* 4 Shipping Categories: Default, Light, Regular and Heavy
-* 3 Shipping Methods (Settings -> Shipping -> Shipping Methods): FedEx, DHL, USPS
-* 2 Stock Locations (Settings -> Shipping -> Stock Locations): New York, Los Angeles
+- 4 Shipping Categories: Default, Light, Regular and Heavy
```

Comment:
Why do we need `default` here? It isn't in the list of categories above.

Reply:
My bad. This is legacy content and shouldn't be diffed at all. This will be fixed in my new shipment setup examples article.